e2e_deployment_files/nestededge_middleLayerBaseDeployment_amqp.json

```diff
@@ -41,7 +41,7 @@
           }
         },
         "modules": {
-          "iotedgeApiProxy": {
+          "IoTEdgeAPIProxy": {
```

Comment:
My understanding is this rename won't have side effects.

Reply:
It won't I am aligning with default name